pkg/volume/aws_ebs/aws_ebs.go

```diff
@@ -421,6 +421,17 @@ type awsElasticBlockStoreProvisioner struct {
 var _ volume.Provisioner = &awsElasticBlockStoreProvisioner{}
 
 func (c *awsElasticBlockStoreProvisioner) Provision() (*v1.PersistentVolume, error) {
+	var fsType string = "ext4"
```

Comment:
this is not specific to this PR but please also make `ext4` a const in `pkg/util/mount` (e.g. DefaultFSType = `ext4`) and reference it elsewhere in provisioner as well as [here](https://github.com/kubernetes/kubernetes/blob/master/pkg/util/mount/mount_linux.go#L359)